test/kubemark/start-kubemark.sh

```diff
@@ -19,10 +19,27 @@
 TMP_ROOT="$(dirname "${BASH_SOURCE}")/../.."
 KUBE_ROOT=$(readlink -e ${TMP_ROOT} 2> /dev/null || perl -MCwd -e 'print Cwd::abs_path shift' ${TMP_ROOT})
 
-source "${KUBE_ROOT}/test/kubemark/common.sh"
 source "${KUBE_ROOT}/test/kubemark/skeleton/util.sh"
 source "${KUBE_ROOT}/test/kubemark/cloud-provider-config.sh"
 source "${KUBE_ROOT}/test/kubemark/${CLOUD_PROVIDER}/util.sh"
+source "${KUBE_ROOT}/cluster/kubemark/${CLOUD_PROVIDER}/config-default.sh"
+source "${KUBE_ROOT}/cluster/kubemark/util.sh"
```

Comment:
Both ```run-e2e-tests.sh``` and ```start-kubemark.sh```  source multiple scripts that get sourced again by ```${KUBE_ROOT}/cluster/kubemark/util.sh```.  We can use ```${KUBE_ROOT}/cluster/kubemark/util.sh``` to source the common scripts between ```run-e2e-tests.sh``` and ```start-kubemark.sh```.